docs/Decentraland Protocol for Explorers.md

```diff
@@ -98,10 +98,18 @@ Used to discover the map around the user. This endpoint is used by the explorer.
 
 ### Interactive layer
 
+#### Chat
+To have a complete social experience Explorers need to support some kind of chat among users. There will be three types of chats, private dms, channels and global. 
+
+Global chat (as in chatting with people around in world) will be supported by the comms service, leveraging the p2p protocol which connects people within a single island (or whatever virtual network available). Everyone is allowed to talk in the global chat, even guests.
+
+The [Matrix protocol](https://matrix.org/) will be leveraged for private and channel support, explorers should connect to a Matrix [homeserver](https://matrix.org/faq/#can-i-write-a-matrix-homeserver%3F) and all interactions will be saved in a single Matrix instance (as expected by the protocol), data will not be shared amongst Matrix instances. So, if a given explorer is connected to a Matrix instance, when switching homeservers, the data will not be synchronized between instances. 
```

Comment:
> So, if a given explorer is connected to a Matrix instance, when switching homeservers, the data will not be synchronized between instances.

Only note here is that I'm not sure this is part of the (Matrix) protocol itself or rather the state of the art with the current implementations, possibly changing in the future. But surely we won't ensure that data replication among Matrix servers will be done.